pkg/xds/bootstrap/parameters.go

```diff
@@ -26,32 +27,32 @@ type AggregateMetricsConfig struct {
 }
 
 type configParameters struct {
-	Id                          string
-	Service                     string
-	AdminAddress                string
-	AdminPort                   uint32
-	ReadinessPort               uint32
-	AppProbeProxyEnabled        bool
-	AdminAccessLogPath          string
-	XdsHost                     string
-	XdsPort                     uint32
-	XdsConnectTimeout           time.Duration
-	Workdir                     string
-	MetricsCertPath             string
-	MetricsKeyPath              string
-	DataplaneToken              string
-	DataplaneTokenPath          string
-	DataplaneResource           string
-	CertBytes                   []byte
-	Version                     *mesh_proto.Version
-	HdsEnabled                  bool
-	DynamicMetadata             map[string]string
-	DNSPort                     uint32
-	ProxyType                   string
-	Features                    []string
-	IsGatewayDataplane          bool
-	Resources                   types.ProxyResources
-	SystemCaPath                string
-	TransparentProxy            *tproxy_config.DataplaneConfig
-	XdsTransportProtocolVariant types.XdsTransportProtocolVariant
+	Id                   string
+	Service              string
+	AdminAddress         string
+	AdminPort            uint32
+	ReadinessPort        uint32
+	AppProbeProxyEnabled bool
+	AdminAccessLogPath   string
+	XdsHost              string
+	XdsPort              uint32
+	XdsConnectTimeout    time.Duration
+	Workdir              string
+	MetricsCertPath      string
+	MetricsKeyPath       string
+	DataplaneToken       string
+	DataplaneTokenPath   string
+	DataplaneResource    string
+	CertBytes            []byte
+	Version              *mesh_proto.Version
+	HdsEnabled           bool
+	DynamicMetadata      map[string]string
+	DNSPort              uint32
+	ProxyType            string
+	Features             xds_types.Features
```

Comment:
make sure to update https://github.com/kumahq/kuma/blob/57a37110fc610c2b718ab71f40d3aca1eff66893/pkg/xds/bootstrap/template_v3.go#L63, as it's no longer a list of strings anymore